include/itkAttenuationImageFilter.hxx

```diff
@@ -91,13 +91,13 @@ void
 AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::BeforeThreadedGenerateData()
 {
   // Verify inputs
-  if (this->GetMaskImage() == nullptr)
+  if (this->GetInputMaskImage() == nullptr)
   {
     itkExceptionMacro("Filter requires a mask image for inclusion estimates!");
   }
   else
   {
-    m_ThreadedMaskImage = this->GetMaskImage();
+    m_ThreadedInputMaskImage = this->GetInputMaskImage();
```

Comment:
I don't think there is a good reason to have the word `threaded` in the name of this iVar. It makes it longer, while not being important.

Reply:
The member is defined solely to reduce repeated calls to `this->GetInputMaskImage()` by threads. I would prefer to keep `threaded` here as a hint as to why we are saving a reference to the input image that we would otherwise get with accessor methods.